documentation/modules/ROOT/partials/assemblies/tutorial/as_viewing-change-events.adoc

```diff
@@ -14,7 +13,7 @@ When you watched the connector start up,
 you saw that events were written to the following topics with the `dbserver1` prefix (the name of the connector):
 
 `dbserver1`::
-The xref:connectors/mysql.adoc#how-the-mysql-connector-handles-schema-change-topics_{context}[schema change topic] to which all of the DDL statements are written.
+The {link-prefix}:{link-mysql-connector}#how-the-mysql-connector-handles-schema-change-topics_{context}[schema change topic] to which all of the DDL statements are written.
```

Comment:
Specifically, why is it using `{link-prefix}` here but the verbatim `xref` elsewhere?

Reply:
Specifically, this xref points to a target that is in a different book downstream (the xref is in the Getting Started Guide and points to a module in the User Guide). By using {link-prefix}, we can use the xref syntax upstream, and swap it for the "link" syntax in the downstream.

I didn't change the other verbatim xrefs because they point to targets that are located in the same page (and therefore will also be valid xrefs in the downstream).

Reply:
Gotcha, thanks!